cvs -z6 diff Makefile.in (in directory E:\Develop\wxWidgets\contrib\src\ogl\)
Index: Makefile.in
===================================================================
RCS file: /pack/cvsroots/wxwidgets/wxWidgets/contrib/src/ogl/Makefile.in,v
retrieving revision 1.36
diff -r1.36 Makefile.in
190a191,193
> COND_MONOLITHIC_0___WXLIB_DEPRECATED_p = \
> 	-lwx_$(PORTNAME)$(WXUNIVNAME)$(WXUNICODEFLAG)$(WXDEBUGFLAG)$(WX_LIB_FLAVOUR)_deprecated-$(WX_RELEASE)$(HOST_SUFFIX)
> @COND_MONOLITHIC_0@__WXLIB_DEPRECATED_p = $(COND_MONOLITHIC_0___WXLIB_DEPRECATED_p)
233c236
< @COND_SHARED_1@	$(SHARED_LD_CXX) $@ $(OGLDLL_OBJECTS) $(LDFLAGS)   -L$(LIBDIRNAME) $(__ogldll___macinstnamecmd) $(__ogldll___importlib) $(__ogldll___soname_flags) $(__ogldll___macver)  $(__WXLIB_MONO_p) $(__LIB_TIFF_p) $(__LIB_JPEG_p) $(__LIB_PNG_p) $(__LIB_ZLIB_p) $(__LIB_ODBC_p) $(__LIB_REGEX_p) $(__LIB_EXPAT_p) $(EXTRALIBS_FOR_BASE) $(EXTRALIBS_FOR_GUI) $(__WXLIB_CORE_p)  $(__WXLIB_BASE_p)
---
> @COND_SHARED_1@	$(SHARED_LD_CXX) $@ $(OGLDLL_OBJECTS) $(LDFLAGS)   -L$(LIBDIRNAME) $(__ogldll___macinstnamecmd) $(__ogldll___importlib) $(__ogldll___soname_flags) $(__ogldll___macver)  $(__WXLIB_MONO_p) $(__LIB_TIFF_p) $(__LIB_JPEG_p) $(__LIB_PNG_p) $(__LIB_ZLIB_p) $(__LIB_ODBC_p) $(__LIB_REGEX_p) $(__LIB_EXPAT_p) $(EXTRALIBS_FOR_BASE) $(EXTRALIBS_FOR_GUI) $(__WXLIB_DEPRECATED_p) $(__WXLIB_CORE_p)  $(__WXLIB_BASE_p)

 	  	 
